sdks/core/src/main/java/org/gbif/pipelines/core/converters/OccurrenceJsonConverter.java

```diff
@@ -200,10 +204,42 @@ private void mapBasicRecord(OccurrenceJsonRecord.Builder builder) {
       JsonConverter.convertVocabularyConcept(gx.getLatestAgeOrHighestStage())
           .ifPresent(gcb::setLatestAgeOrHighestStage);
 
+      GeologicalRange.Builder rangeBuilder = GeologicalRange.newBuilder();
```

Comment:
Can it be the case when START_AGE_TAG and START_AGE_TAG have different types? Fx:
START_AGE_TAG - getEarliestAgeOrLowestStage
END_AGE_TAG - getLatestPeriodOrHighestSystem

Reply:
I actually did it on purpose so if, for example, the earliest is provided but not the latest I take the latest from the parent but now I'm realising that it makes more sense to just take both start and end from the term provided(in case there is only one), the earliest in this example.

So I'll change it